data/campaigns/Northern_Rebirth/scenarios/12a_Get_the_Gold.cfg

```diff
@@ -23,74 +23,82 @@
     [story]
         [part]
             # wmllint: local spelling Angthurim
-            story= _ "After reducing Castle Angthurim, the party set off after Rakshas, hoping that Hidel’s elves had been able to hold him."
+            story= _ "After conquering Castle Angthurim, the party set off after Rakshas, hoping that Hidel’s elves had been able to hold him."
             music=sad.ogg
         [/part]
         [part]
             story= _ "Orcs are heavy-footed creatures; Rakshas’s trail was readily followed. But an ominous silence, broken only by the cawing of ravens, brooded over the road he had taken."
         [/part]
         [part]
-            story= _ "Soon they encountered the wrack of a great battle. Bodies of elves and orcs lay everywhere. Broken weapons and smashed armor were strewn about in profusion. Elvish arrows and orcish crossbow bolts bristled from the nearby trees. The ground was splotched red with blood."
+            story= _ "Soon they encountered the wrack of a great battle. Bodies of elves and orcs lay everywhere. Broken weapons and smashed armor were strewn about and elvish arrows and orcish bolts bristled from the nearby trees. The ground was red with blood."
```

Comment:
How about 'The ground was stained red with blood'?

Also, I notice here and in other places (including your other PRs) a new line is introduced before the ending `[/part]` tag. It's not a big deal, but I think it would be good to retain the original presentation for consistency.

Reply:
I didn't see this place, so I'm going to make this change.